File: 06_Day/Review.js
function even_odd_indexed (test){
    let even = "" 
    let odd = ""
    for (let index = 0; index < test.length; index++) {
        index%2==0 ? even +=test[index] : odd += test[index]
    }
    console.log(even + " " + odd)
}


function processData(input) {
    //Enter your code here
    const testCases = input.split('\n').slice(1);
    for (const test of testCases) {
        even_odd_indexed(test)
    }
} 

process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
});

process.stdin.on("end", function () {
   processData(_input);
});

File: 06_Day/Review.py
# Enter your code here. Read input from STDIN. Print output to STDOUT
import sys
data = [line.rstrip().split()[0] for line in sys.stdin.readlines()]

def even_odd_indexed(string):
    even_indexed = ''
    odd_indexed = ''
    for i in range(0, len(string)):
        if i%2 == 0:
            even_indexed+=string[i]
        else:
            odd_indexed+=string[i]
    print(even_indexed + ' ' + odd_indexed)


test_cases = data[1:]
for test in test_cases:
    even_odd_indexed(test)



